### ISSUE TITLE
Steam Big Picture doesn't launch certain games. 

### ISSUE THREAD

**Issue by Lehowl** [moonlight-stream/moonlight-android#166]
Ok, I have a problem that is driving me insane. I set up a Raspberry PI with Moonlight Embedded last September and I've noticed some issues - as in some games would work perfectly whereas some wouldn't launch through steam big picture - and didn't have the will to track down and fix them. However, as I'm in vacation right now I decided start from the beginning and installed a fresh Raspbian Jessie and the the moonlight. However the same issues happened again and I've finally understood how to reproduce them. After launching a game that is recognized as a Gamestream ready game(for example Rocket League)  I can't launch another game that isn't Gamestream ready(for example Cities Skyline) through the Steam Big Picture. **I've installed Moonlight on my Android Phone(Xperia Z2 with Lollipop) and the same issue happened.** 

I couldn't find any similar issues reported or a fix for it which wonders me if this is only a problem happening in my PC. I deactivated all services and unecessary boot programs through msconfig and task manager to pinpoint if any  3rd party program had an incompatibility without success. 

**TL;DR: After launching Steam Big Picture I can play any game I want. However, after launching a Gamestream ready game through Steam Big Picture, I can't launch a non-Gamestream ready through Steam Big Picture without rebooting Steam application first.**

Thus I'm asking: is this a known issue? Anyone besides me can reproduce it? Is it an issue from Geforce experience(which I'm inclined to believe) side or from moonlight?

My host setup is:
Windows 10
Nvidia GTX 970
Geforce Experience 2.9.1.35
Nvidia Driver 361.91(this happened before back in September as I said, then I don't believe is a new driver issue)

Both Moonlight Android and Embedded don't work.

Thank you!

**Comment by Lehowl** [comment]
I think I've found the cause. I was checking the logs generated by Geforce experience in "C:\Users\<myusername>\AppData\Local\NVIDIA\NvBackend" and noticed some fails regarding the app trying to optimize games without support. I disabled the automatic optimization and all games appears to be working fine now in my Android phone, I just need to confirm that in Pi will alse be fine, but I will only be able to do this later.

Does this happen with anyone else?  Or is this a bug specific to my computer?

Best wishes
